README.md

```diff
@@ -1,8 +1,126 @@
+
+# Motivation
+
+We are 7 computer science and communication system students at EPFL, currently enrolled in the Software Engineering course CS-305. We are allowed to create any application of our choice that fullfill following pre-requisites:
+
+- Split app model: firebase
+- Phone sensors: GPS
+- User support
+- Database access
+- Automated test suite: 80% code coverage
+
+# Build Status
 
 [![Build Status](https://travis-ci.org/basbeu/theSofties.svg?branch=master)](https://travis-ci.org/basbeu/theSofties)
 
 <a href="https://codeclimate.com/github/basbeu/theSofties/maintainability"><img src="https://api.codeclimate.com/v1/badges/ca33d5d2d2e82d260e05/maintainability" /></a>
 
 <a href="https://codeclimate.com/github/basbeu/theSofties/test_coverage"><img src="https://api.codeclimate.com/v1/badges/ca33d5d2d2e82d260e05/test_coverage" /></a>
 
+# Features
+
+######Completed
+
+- users can create accounts, login, edit their profile and reset their password
+- users can see a list of favors, a detailed description of a favor and can ask for a new one and set a deadline
+- there are several categories favors can be ordered upon, additionally they can be ordered based on location, most recent and searched by name or description
+
+###### Roadmap
+
+- users can add a profile picture and add media files to their favors
+- users can see the favors on a map around them with a small pin and a tag
+- users can chat about meeting up for completing an issue and receive notifications
+- users can "pay" using tokens and mark a favor as completed
+- users can request multiple people for a favor
+- users can see a history of the favors they have asked for and completed
+
+# Scrum
+
+### Sprint 1
+
+##### Scrum Master: Jeremy, presenter: Bastien
+
+- git setup, filling backlog, authentification and rudimentary database and UI
+
+### Sprint 2
+
+##### Scrum Master: Nicolas, presenter: Andrea
+
+- build a professional and concise database structure with firestore
+- expand UI & profile functionality
+- added basic favor capability that communicated with firestore using observables
+
+<img src="https://mail.sanchez.at/files/favor-screen-1/favor-screen-1.png" alt="drawing" width="200"/>
+
+### Sprint 3
+
+##### Scrum master: Charline, presenter: Jeniffer
+
+- favor list with UI for searching & sorting, as well as asking for a new favor
+- location, distance, category, etc. are recorded alongside favor
+
+
+
+# Tests
+
+right now there is only a very limited amount of tests
+
+This will be changed in Sprint 3
+
+``` bash
+code for running the single line command in the command line goes here
```

Comment:
What do you mean with a single line command?

Reply:
Automated test suite: Your app must have from day one a test suite that can be executed by running a single command on the command line. You must maintain a minimum level of 80% code coverage.
http://dslab.epfl.ch/teaching/sweng/proj

Reply:
Also please don't block the PR if you don't understand something, that is why there is a 3. button called, comment but approve